Formula/counterfeiter.rb

```diff
@@ -4,7 +4,7 @@ class Counterfeiter < Formula
   url "https://github.com/maxbrunsfeld/counterfeiter/archive/refs/tags/v6.5.0.tar.gz"
```

Comment:
Please update to https://github.com/maxbrunsfeld/counterfeiter/archive/refs/tags/v6.6.1.tar.gz

Reply:
Doing this in #122307